prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/TranslationFunctions.kt

```diff
@@ -0,0 +1,67 @@
+package gov.cdc.prime.router.fhirengine.translation.hl7.utils
+
+import ca.uhn.fhir.model.api.TemporalPrecisionEnum
+import ca.uhn.hl7v2.model.v251.datatype.DTM
+import org.hl7.fhir.r4.model.BaseDateTimeType
+
+/**
+ * This interface contains the required method signatures required to implement custom FHIR functions
+ */
+interface TranslationFunctions {
```

Comment:
Comments in here to be updated this interface isn't used for custom FHIR functions.